src/module/iotjs_module_process.c

```diff
@@ -39,7 +39,6 @@ static iotjs_jval_t WrapEval(const char* source, size_t length, bool* throws) {
   int len1 = strlen(wrapper[1]);
 
   iotjs_string_t code = iotjs_string_create("");
-  iotjs_string_reserve(&code, len0 + length + len1);
```

Comment:
It wouldn't be a bad idea if we could create a string with a specific size, and use some iotjs_string_rewrite function which can overwrite part of a string. So we would still have a single memory allocation in such cases.